includes/polling/applications/unbound.inc.php

```diff
@@ -49,14 +53,18 @@
     'null' => $unbound['num.query.type.NULL'],
     'wks' => $unbound['num.query.type.WKS'],
```

Comment:
have you tried to change those keys only to uppercase instead of changing other stuff?

Reply:
I can't test that atm as I migrated to docker install, which is why I made the PR in the first place so that it hopefully can get fixed in the master and pulled into the docker image as I all but forgot about the issue seeing as I had it fixed on the previous install.

Keeping the data would be preferable of course, I'll spin up a new librenms install and test next time I find few free hours, which might not be until next weekend.

Reply:
It should be enough to just upper case these array keys.

Reply:
I've created new ubuntu 20.04 vm, installed librenms and had it monitor the router. The queries graph was empty as expected. then I changed the 2 lines to: 
'NULL' => $unbound['num.query.type.NULL'],
'WKS' => $unbound['num.query.type.WKS'],
(disclaimer: I have no idea what I'm doing, hopefully this was what you meant by changing these 2 to uppercase)

This changed nothing, on top of which the queries.rrd doesn't get created even if I deleted the old one.

Reply:
i mean all Keys from line 44 - 61, not only those 2 ;-)

Reply:
Ok, changed the lines 44 - 61 like so:
```
    'TYPE0' => $unbound['num.query.type.TYPE0'],
    'A' => $unbound['num.query.type.A'],
    'NS' => $unbound['num.query.type.NS'],
    'CNAME' => $unbound['num.query.type.CNAME'],
    'SOA' => $unbound['num.query.type.SOA'],
    'NULL' => $unbound['num.query.type.NULL'],
    'WKS' => $unbound['num.query.type.WKS'],
    'PTR' => $unbound['num.query.type.PTR'],
    'MX' => $unbound['num.query.type.MX'],
    'TXT' => $unbound['num.query.type.TXT'],
    'AAAA' => $unbound['num.query.type.AAAA'],
    'SRV' => $unbound['num.query.type.SRV'],
    'NAPTR' => $unbound['num.query.type.NAPTR'],
    'DS' => $unbound['num.query.type.DS'],
    'DNSKEY' => $unbound['num.query.type.DNSKEY'],
    'SPF' => $unbound['num.query.type.SPF'],
    'ANY' => $unbound['num.query.type.ANY'],
    'OTHER' => $unbound['num.query.type.other']
```
The queries rrd remained unchanged next polling cycle. I then deleted it but the new one was the same.

Reply:
Also, I was thinking about this: It is of no consequence if the old rrd needs to be deleted, there is no point in trying to come up with a fix to preserve the data, as there is nothing to preserve, the unbound application does not work at all for queries thus there is nothing to preserve.